### PR TITLE
CPM-984: Limit conditions to 10 items

### PR DESCRIPTION
**Pull request by pierallard**

### CODE REVIEW

**Review by Beninho — APPROVED** [review]

File: components/identifier-generator/front/src/feature/tabs/conditions/AddConditionButton.tsx [hunk 1]
```diff
@@ -59,7 +61,14 @@ const AddConditionButton: React.FC<AddConditionButtonProps> = ({conditions, onAd
 
   return (
     <Dropdown>
-      <Button active ghost level="secondary" onClick={addElement} size="small">
```
Comment (cemorin) [inline components/identifier-generator/front/src/feature/tabs/conditions/AddConditionButton.tsx:62]: For the properties, we hide the button. Maybe we should do the same here?

File: components/identifier-generator/back/tests/features/createIdentifierGenerator.feature [hunk 2]
```diff
@@ -177,6 +177,11 @@ Feature: Create Identifier Generator
     Then I should get an error with message 'conditions[0][type]: Type "unknown" can only be one of the following: "enabled"'
     And the identifier should not be created
 
+  Scenario: Cannot create an identifier generator with too much conditions
```
Comment (mmetayer) [inline components/identifier-generator/back/tests/features/createIdentifierGenerator.feature:180]:
```suggestion
  Scenario: Cannot create an identifier generator with too many conditions
```

File: components/identifier-generator/back/tests/features/updateIdentifierGenerator.feature [hunk 3]
```diff
@@ -148,6 +148,10 @@ Feature: Update Identifier Generator
     When I try to update an identifier generator with unknown condition type
     Then I should get an error on update with message 'conditions[0][type]: Type "unknown" can only be one of the following: "enabled"'
 
+  Scenario: Cannot update an identifier generator with too much conditions
```
Comment (mmetayer) [inline components/identifier-generator/back/tests/features/updateIdentifierGenerator.feature:151]:
```suggestion
  Scenario: Cannot update an identifier generator with too many conditions
```

File: components/identifier-generator/front/src/feature/tabs/conditions/AddConditionButton.tsx (hunk 1 above)
Comment (mmetayer) [reply components/identifier-generator/front/src/feature/tabs/conditions/AddConditionButton.tsx:62]: to decide with PM/UX (my personal preference would be to disable the button in the properties tab as well)

**Review by cemorin — APPROVED** [review]

**Pull request merged**